command.py

```diff
@@ -83,9 +83,33 @@ def handle_mode(state: mantatail.ServerState, user: mantatail.UserConnection, mo
         process_user_modes()
 
 
-# !Not implemented
-def _handle_kick(message: str) -> None:
-    pass
+def handle_kick(state: mantatail.ServerState, user: mantatail.UserConnection, arg: str) -> None:
+    args = arg.split()
```

Comment:
If the kick reason contains repeated spaces, this will treat each repeated space as if it was just a single space. Not a big deal if
```
you are  a    troll
```
turns into
```
you are a troll
```
but we might as well fix this.

```suggestion
    args = arg.split(" ")
```